pyproject.toml

```diff
@@ -36,7 +36,6 @@ antsibull-lint = "antsibull.cli.antsibull_lint:main"
 python = "^3.6.1"
 antsibull-changelog = ">= 0.14.0"
 antsibull-core = ">= 1.3.0, < 2.0.0"
-antsibull-docs = ">= 1.0.0, < 2.0.0"
```

Comment:
I guess it's correct to remove the `antsibull-docs` dependency if nothing in here imports it, but this is also somewhat of a breaking change, because `pip install antsibull` will no longer include the `antsibull-docs` binary.

Reply:
I adjusted the changelog fragment in b232463fdd6047a7d075755fd50bc71dc351dd15. The feature removals are also breaking changes, so one more doesn't matter anyway ;)